### ISSUE TITLE
BoxScaleLoss is nan when training YOLOv3 on my own dataset

### ISSUE THREAD

**Issue by chinagaodali** [dmlc/gluon-cv#684]
> Namespace(batch_size=10, data_shape=416, dataset='coco', epochs=200, gpus='0', label_smooth=False, log_interval=100, lr=0.0001, lr_decay=0.1, lr_decay_epoch='160,180', lr_decay_period=0, lr_mode='step', mixup=False, momentum=0.9, network='darknet53', no_mixup_epochs=20, no_random_shape=True, no_wd=False, num_samples=781, num_workers=4, resume='', save_interval=10, save_prefix='./backup/yolo3_darknet53_coco', seed=233, start_epoch=0, syncbn=False, val_interval=1, warmup_epochs=0, warmup_lr=0, wd=0.0005)
> Start training from [Epoch 0]

...

> [Epoch 197] Training cost: 60.367, ObjLoss=9.816, BoxCenterLoss=11.158, BoxScaleLoss=nan, ClassLoss=2.839

From Epoch 197 to the last Epoch,BoxCenterLoss=nan. So,why only the  BoxScaleLoss=nan but other loss is normal? Should I just stop the training?

**Comment by zhreshold** [comment]
You should stop, it won't recover from nan, you better check if other hyperparameters is good
